cider/include/cider/CiderAllocator.h

```diff
@@ -86,6 +94,7 @@ class AlignAllocator : public CiderAllocator {
   }
 
   void deallocate(int8_t* p, size_t size) final {
+    std::cout << "deallocate memory" << size << std::endl;
```

Comment:
please avoid using `std::cout`.

Reply:
Sorry，Accidentally submitted code for local debug lol